text/0010-burn-coretime-revenue.md

```diff
@@ -0,0 +1,33 @@
+# RFC-0010: Burn Coretime Revenue
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | 19.07.2023                                                                   |
+| **Description** | Revenue from Coretime sales should be burned                                                                   |
+| **Authors**     |                   Jonas Gehrlein                                                                          |
+
+## Summary
+
+The Polkadot UC will generate revenue from the sale of available Coretime. The question then arises: how should we handle these revenues? Broadly, there are two reasonable paths – burning the revenue and thereby removing it from total issuance or divert it to the Treasury. This Request for Comment (RFC) presents arguments favoring burning as the preferred mechanism for handling revenues from Coretime sales.
+
+## Motivation
+
+How to handle the revenue accrued from Coretime sales is an important economic question that influences the value of DOT and should be properly discussed before deciding for either of the options. Now is the best time to start this discussion.
+
+## Stakeholders
+
+Polkadot DOT token holders.
+
+## Explanation
+
+This RFC discusses potential benefits of burning the revenue accrued from Coretime sales instead of diverting them to Treasury. Here are the following arguments for it.
+
+It's in the interest of the Polkadot community to have a consistent and predictable Treasury income, because volatility in the inflow can be damaging, especially in situations when it is insufficient. As such, this RFC operates under the presumption of a steady and sustainable Treasury income flow, which is crucial for the Polkadot community's stability. The assurance of a predictable Treasury income, as outlined in a prior discussion [here](https://forum.polkadot.network/t/adjusting-the-current-inflation-model-to-sustain-treasury-inflow/3301), or through other equally effective measures, serves as a baseline assumption for this argument. 
```

Comment:
this basically is saying we cannot approve this RFC without the inflation model adjustment RFC is accepted

Reply:
Well, not necessarily. Even without the change proposed in the forum post, I doubt that coretime revenues can become a reliable funding source for the Treasury in the short-/midterm. So, funding needs to come from other sources. Either we further depend on the staking inefficiency or we explicitly fix it with the proposed mechanism.

With that in mind, my argument is that Treasury funding needs to be fixed in some other dimension and then it's better to burn the revenue based on all the other arguments presented. But, this point is particularly strong once we fix Treasury inflow with the mechanism described in the forum post.

Reply:
Anyway they're both linked.  Alistair and Jonas have a combined argument here that burning here and inflation adjustment together make sense. 

At a high level, this sort of thing is almost universal in crypto currency.  Bitcoin shall eventually become insecure by not doing this sort of thing https://economics.princeton.edu/working-papers/on-the-instability-of-bitcoin-without-the-block-reward/#  

Really, the onus is upon "fees pay" people to argue that fees paying does not fuck up anything else, otherwise we should do this sort of thing as a default.  Of course we might need to link both this and the treasury change, just to simplify the governance process, not sure.